aegis_description/README.md

```diff
@@ -12,3 +12,109 @@ ros2 launch aegis_description display.launch.py
 ```
 
 ![aegis_preview](./docs/aegis_preview.png)
+
+## Configuration files
```

Comment:
Extended this section with  the purpose of the `launch` files, `rviz` config and `urdf` catalog